pyfarm/models/project.py

```diff
@@ -1,73 +0,0 @@
-# No shebang line, this module is meant to be imported
```

Comment:
Little curious why this is being removed.  Is it because this is not really fully implemented or because you ended up not finding a need for it.  I'm guessing it's a little of both, but mostly because of the implementation.